nRf52840peripheral/prj.conf

```diff
@@ -14,29 +14,26 @@ CONFIG_LOG=y
 CONFIG_PM=y
 CONFIG_PM_DEVICE=y
 #CONFIG_TRACING_SYSCALL=y
+
 #For enabling the definitionns of MTU exchange functions
 CONFIG_BT_GATT_CLIENT=y
-
+CONFIG_BT_BUF_EVT_RX_COUNT=5
```

Comment:
what is the significance of this one ?

Reply:
this is for handling multiple incoming HCI events from the
Controller. This was in the main branch, I merged it with my branch Wifi Configuration.